tests/renderer/components/__snapshots__/commands-version-chooser-spec.tsx.snap

```diff
@@ -1,45 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
This is me being ignorant of RTL - do we need to generate new snapshots for RTL to test against? I think RTL still uses snapshot testing, unless I'm mistaken here? Reading through: https://testing-library.com/docs/react-testing-library/migrate-from-enzyme

Reply:
We could generate new snapshots, but from what I gather from the RFC @erickzhao linked in https://github.com/electron/fiddle/issues/1408, and I might be wrong here, we probably shouldn't:

> Most PRs to the React components end up regenerating the snapshot. We don’t actually pay attention to the diff too much during development, so these changes end up being low signal and high noise.